ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx

```diff
@@ -63,10 +63,22 @@ interface VariableEditFormProps {
   initialVariableDefinition: VariableDefinition;
   onChange: (def: VariableDefinition) => void;
   onCancel: () => void;
+  isNameReadonly?: boolean;
```

Comment:
isCreating, isReadonly, saveButtonLabel, title. We could simplify having an action string with a finite number of possibilities ``'create' | 'edit' | 'view'``

For example, the following would just display the var
```html
<VariableForm action="view" initialVariableDefinition={myVariable}>
</VariableForm
```